test/powershell/Modules/Microsoft.PowerShell.Management/Resolve-Path.Tests.ps1

```diff
@@ -23,28 +42,11 @@ Describe "Resolve-Path returns proper path" -Tag "CI" {
         $result = Resolve-Path -LiteralPath "TestDrive:\\\\\"
         ($result.Path.TrimEnd('/\')) | Should Be "TestDrive:"
     }
-    It "Resolve-Path -Relative should return correct path on different drive" {
-        $base = Join-Path $TestDrive "ResolvePath.relative"
-        $root = Join-Path $base "fakeroot"
-        $file = Join-Path $root "file.txt"
-        $expectedFilePath = Join-Path "." "fakeroot" "file.txt"
-        $driveName = "RvpaTest"
-        $null = New-Item -Path $base -ItemType Directory -Force
-        $null = New-Item -Path $root -ItemType Directory -Force
-        $null = New-Item -Path $file -ItemType File -Force
-        $null = New-PSDrive -Name $driveName -PSProvider FileSystem -Root $root
-        $driveRoot = Join-Path "$driveName`:" ""
-        $driveFile = Join-Path "$driveName`:" "file.txt"
-        try {
-            Push-Location -Path $driveRoot
-            Resolve-Path -Path $base -Relative | Should BeExactly $base
-        }
-        finally {
-            Pop-Location
-        }
+    It "Resolve-Path -Relative should return correct path on different drive" -TestCases $relCases {
```

Comment:
Add a item for test name in `$relCases` which can be used in the `It`. Otherwise, all the variations have the same name in the log file.

It "Resolve-Path -Relative should return correct path on different drive - <testname>" -TestCases $relCases